docs/bsp.md

```diff
@@ -1288,6 +1288,10 @@ files:
 Build tools can decide where to write the connection files depending on the
 specific implementation details so long as the previous requirements are met.
 
+The `$XDG_DATA_DIRS` is defined in the [XDG Base Directory
+Specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-0.6.html).
+For Windows systems, the global location must be `%APPDATA%/.bsp/`.
```

Comment:
why not also `%PROGRAMDATA%\jupyter\kernels`?

Reply:
Can we maybe have a pretty table like kernelspec? 😇 

<img width="996" alt="screen shot 2018-10-28 at 09 09 28" src="https://user-images.githubusercontent.com/1408093/47613533-3fc83a80-da91-11e8-9c2a-83de86f720a7.png">